nextflow.config

```diff
@@ -24,6 +25,9 @@ params {
     // Default to signed request when accessing s3 bucket
     // can be overwritten using params.aws_no_sign_request
     aws_no_sign_request = false
+
+    //To match samtools arg number of threads with no. of cpus
+    samtools_threads = '4'
```

Comment:
This should be a parameter extract from nextflow resource requirements, not hard coded

Reply:
Changed to argument..
```
        publishDir = [
            path: "${params.publishdir}/samtools",
```
```
nextflow run \
../../main.nf -config \
../../nextflow.config \
-params-file params.yml  \
-work-dir ./work  \
--publish_dir ./output \
--samtools_threads 2
```

Reply:
Added ${task.cpus}